core/string/ustring.cpp

```diff
@@ -939,6 +939,7 @@ const char32_t *String::get_data() const {
 }
 
 void String::erase(int p_pos, int p_chars) {
+	p_pos = MAX(p_pos, 0);
```

Comment:
Should it fail on negative indices instead?

Reply:
I could change it to fail, I just wanted to preserve the previous behavior.